src/emulator.py

```diff
@@ -428,10 +428,27 @@ def apply_settings(
 
 def allow_unsafe() -> None:
     client = TrezorClientDebugLink(get_device())
-    # ignore for Legacy firmware, there is no such setting
-    if client.features.major_version == 1:
-        return
     client.open()
     time.sleep(SLEEP)
-    device.apply_settings(client, safety_checks=1)  # TODO
+
+    # Each model has its own supported safety checks level
```

Comment:
To be precise T1 does not support PromptAlways, TT supports all three of them (though not all of them are in Suite).

Reply:
Is it alright just to replace this comment by `T1 does not support PromptAlways`?

Reply:
Done in 27200b1